embark.el

```diff
@@ -955,25 +955,32 @@ the minibuffer is open, the message is added to the prompt."
 Besides the bindings in KEYMAP, the user is free to use all their
 key bindings and even \\[execute-extended-command] to select a command.
 UPDATE is the indicator update function."
-  (let* ((key (let ((overriding-terminal-local-map keymap))
-                (embark--read-key-sequence update)))
+  (let* ((keys (let ((overriding-terminal-local-map keymap))
+                 (embark--read-key-sequence update)))
          (cmd (let ((overriding-terminal-local-map keymap))
-                (key-binding key 'accept-default))))
+                (key-binding keys 'accept-default))))
     (pcase cmd
-      ('embark-keymap-help
+      ((or 'embark-keymap-help
```

Comment:
I wonder why it is even necessary to check `embark-keymap-help` here. Should this not be subsumed automatically by the more general logic here?

We should probably also not bind `embark-keymap-help` in the `embark-meta-map`.

Reply:
Ideally we should probably remove all special casing of `embark-keymap-help` and replace it with `help-char` checks as you have it here.

Reply:
Yes, I was thinking this too, but there's still a reason to have something like `embark-keymap-help`. The emacs command loop, as well as this, shows help only if `help-char` is not otherwise currently bound. By default, `C-h` is bound to `help-map`, so without special treatment, `C-. C-h` would go into the help prefix commands instead of showing help for the embark action map.

So there needs to be some way of signaling that we want a key to show embark help, rather than behave just as it does outside embark.

Reply:
I see, you are right. The issue is that for `C-. C-h` the keymap prompter enters the global prefix map `help-map` and does not return immediately. We need `embark-keymap-help` to return immediately. Thanks for the explanation!